bin/pycbc_multi_inspiral

```diff
@@ -241,26 +241,27 @@ def coherent_snr(snr_triggers, index, threshold, projection_matrix,
     rho_coh = rho_coh[rho_coh > threshold]
     return rho_coh, index, snrv, coinc_snr
 
-def coincident_snr(snr_dict, index, threshold):
+def coincident_snr(snr_dict, index, threshold, time_delay):
     """
     Output: rho_coinc: Coincident snr triggers
             index    : The subset of input index that survive the cuts
             coinc_triggers: Dictionary of individual detector SNRs at
                             indexes that survive cuts
-    Input: snr_dict: Dictionary of individual detector SNRs in geocent time
+    Input: snr_dict: Dictionary of individual detector SNRs
            index   : Geocent indexes you want to find coinc SNR for
            threshold: Indexes with coinc SNR below this threshold are cut
+           time_delay: Dictionary of time delay for each detector
```

Comment:
Is this time_delay (in seconds) or time_delay (as an index). Maybe calling this time_delay_idx would make it clear, but can you also specify this here. I note that the formatting you are using here is not `numpydoc` format, so this would not correctly display in the online docs (although functions in executables don't get parsed anyway). You can see a bunch of examples in any of the modules under `pycbc` (or in numpy itself) for what this looks like.

.... We should also consider moving a bunch of these functions into some module and not in the executable, but that's a different PR.

Reply:
Thanks for the feedback! I'll clarify here that it is time delay as an index. Because all of the functions in this executable are documented in this format, I think I'll leave that change for another time if that's acceptable.